docs/_config.yml

```diff
@@ -0,0 +1,24 @@
+title: Falafel
+# email: your-email@example.com
+description: >- # this means to ignore newlines until "baseurl:"
+  A Node.js framework for building Tray.io connectors
+baseurl: "/falafel" # the subpath of your site, e.g. /blog
+url: "https://trayio.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+
+remote_theme: pmarsceill/just-the-docs
+theme: "just-the-docs"
+
+# logo: "https://tray.io/favicon-32x32.png"
+
+color_scheme: light

```

Comment:
Wot, no dark?

Reply:
also, stuff gets a bit hard to read on the default dark theme 
![image](https://user-images.githubusercontent.com/14856227/135097282-bd17af11-ec36-4287-896f-6c21c15d8aa5.png)

Reply:
Uh, yeah, that's not good. There probably is a nice dark theme somewhere. The toggle would be good if it's not too much effort.

Reply:
Yeah im trying to keep this slim on complex stuff. currently its just a config file and some markdown files, so its incredibly easy for us to maintain. I want to avoid falling down any rabbit holes that would make it hard to document things

Reply:
I am currently in a light mode phase so quite enjoy the light mode